### PR TITLE
moderation notifications for sdiywiki

### PR DESCRIPTION
**Pull request by robkam**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -3588,6 +3588,19 @@
...
+	'wgModerationNotificationEnable' => array( // Enable or disable notifications. 
+		'default' => false,
+		'sdiywiki' => true,
+	),
+	'wgModerationNotificationNewOnly' => array( // Notify administrator only about new pages requests. 
+		'default' => false,
+	),
+	'wgModerationEmail' => array( // Email to send notifications. 
+		'default' => 'wgEmergencyContact',
+		// 'sdiywiki' => 'admin@sdiy.info',
```
Comment (Reception123) [inline LocalSettings.php]: There isn't really a point of leaving this commented out..

**Comment by Reception123** [comment]
Sorry if I wasn't clear enough. John's comment here: https://github.com/miraheze/mw-config/pull/1864#issuecomment-346608015 still stands

**Comment by robkam** [comment]
I don't understand. The default wgEmergencyContact is deprecated https://www.mediawiki.org/wiki/Manual:$wgEmergencyContact in favour of wgPasswordSender, but that is set to noreply@miraheze.org I'll never see the email notifications?

**Comment by Reception123** [comment]
Hmm, I am slightly confused myself in this regard. Maybe @JohnFLewis could clarify his comment?

**Comment by JohnFLewis** [comment]
Documentation is very very poor. MediaWiki docs suggests it's the email it is sent FROM but the code docs say it's the email it is sent TO. If it's sent TO, then it's fine but my understanding was it was the FROM.

**Comment by robkam** [comment]
In preferences I have an email address and set for individual notifications as they come in but I get no notifications from the moderation extension. I need to fix that somehow.

**Comment by robkam** [comment]
added 'sdiywiki' => 'admin@sdiy.info',

**Comment by Reception123** [comment]
Well looks like it's TO, according to https://github.com/edwardspec/mediawiki-moderation/blob/41459a9ba4ee020579596ae2f867685a04859651/hooks/ModerationEditHooks.php#L199

**Comment by Reception123** [comment]
So, it should be good.

File: LocalSettings.php [hunk 2]
```diff
@@ -3588,6 +3588,19 @@
...
+	'wgModerationNotificationEnable' => array( // Enable or disable notifications. 
+		'default' => false,
+		'sdiywiki' => true,
+	),
+	'wgModerationNotificationNewOnly' => array( // Notify administrator only about new pages requests. 
+		'default' => false,
+	),
+	'wgModerationEmail' => array( // Email to send notifications to. 
+		'default' => 'wgEmergencyContact',
+		 'sdiywiki' => 'admin@sdiy.info',
```
Comment (Reception123) [inline LocalSettings.php]: Minor but please fix this extra space before 'sdiywiki'

**Pull request merged**